spec/features/browse_dashboard_files_spec.rb

```diff
@@ -63,10 +63,11 @@
       end
     end
 
-    it "should allow me to edit works in batches", skip: 'Not yet implemented' do
+    it "should allow me to delete works in batches",  :js => true do
```

Comment:
@jcoyne we are not planning to allow for batch edit of works at the moment, so I though no reason to leave the test hanging around. What part of this line is @mjgiarlo going to comment on?